src/ASM/LR/ASMu3D.h

```diff
@@ -16,6 +16,7 @@
 
 #include "ASMunstruct.h"
 #include "ASM3D.h"
+#include "LRSpline/LRSpline.h"
```

Comment:
Do we actually need to include this header file here? Cannot see it is used here. Isn't it sufficient to forward-declare LRSplineVolume instead?